src/flint/gfp_mat.jl

```diff
@@ -508,3 +508,22 @@ function matrix_space(R::fpField, r::Int, c::Int; cached::Bool = true)
    # TODO/FIXME: `cached` is ignored and only exists for backwards compatibility
   fpMatrixSpace(R, r, c)
 end
+
+################################################################################
+#
+#  Kernel
+#
+################################################################################
+
+function AbstractAlgebra.Solve.kernel(A::fpMatrix; side::Symbol = :right)
+   AbstractAlgebra.Solve.check_option(side, [:right, :left], "side")
+
+   if side === :left
+      K = AbstractAlgebra.Solve.kernel(A, side = :right)
+      return transpose(K)
+   end
+
+   K = zero_matrix(base_ring(A), ncols(A), max(nrows(A), ncols(A)))
+   n = ccall((:nmod_mat_nullspace, libflint), Int, (Ref{fpMatrix}, Ref{fpMatrix}), K, A)
```

Comment:
I assume no, but could this also be used for `FqMatrix` if the characteristic is "right"?

Reply:
As discused, there is `fq_mat_nullspace`, `fq_nmod_mat_nullspace` and `fq_default_nullspace` and they should be wrapped.